batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/service/StudentService.java

```diff
@@ -5,12 +5,16 @@
 import com.lftechnology.batch7crud.dao.StudentDAO;
 import com.lftechnology.batch7crud.entity.Student;
 import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.exception.ValidationExceptions;
+import com.lftechnology.batch7crud.validator.StudentValidator;
 
 public class StudentService {
   private StudentDAO studentDAO = new StudentDAO();
 
-  public void insert(Student student) throws DataException {
-    studentDAO.insert(student);
+  public void insert(Student student) throws DataException, ValidationExceptions {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)